lib/ui/view/channel/new_channel/new_channel.dart

```diff
@@ -57,54 +70,202 @@ class _NewChannelState extends State<NewChannel> {
+                        hintText: 'Add a Desription',
+                        hintStyle: AppTextStyles.body2Medium.copyWith(
+                          color: AppColors.zuriDarkGrey,
+                        ),
+                      ),
                     ),
                   ),
                   const Divider(
-                    thickness: 2.0,
+                    thickness: 0.5,
+                    color: AppColors.greyColor,
+                  ),
+                  UIHelper.verticalSpaceMedium,
+                  const Divider(
+                    thickness: 0.5,
+                    color: AppColors.navBarItemColor,
+                  ),
+                  Container(
+                    padding: EdgeInsets.only(left: 16.0, bottom: 14.0),
+                    child: Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: [
+                        Text(
+                          'Make channel private',
+                          style: AppTextStyles.heading9.copyWith(
+                            letterSpacing: 0.5,
+                            fontWeight: FontWeight.w400,
+                            height: 2.0,
+                          ),
+                        ),
+                        Row(
+                          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                          children: [
+                            Text(
+                              'A private channel can only be viewed or joined',
+                              style: AppTextStyles.body2Medium,
+                            ),
+                            SizedBox(
+                              height: 20,
+                              child: Switch(
+                                inactiveTrackColor: Color(0XFFEBEBEB),
+                                activeColor: AppColors.zuriPrimaryColor,
+                                value: _value,
+                                onChanged: (val) {
+                                  setState(() {
+                                    _value = val;
+                                  });
+                                },
+                              ),
+                            )
+                          ],
+                        ),
+                        //add this part here because the Row widget won't align with just the upper part of the text when there's a new line
+                        Text(
+                          'by invitation.',
+                          style: AppTextStyles.body2Medium,
+                        )
+                      ],
+                    ),
+                  ),
+                  const Divider(
+                    thickness: 0.5,
+                    color: AppColors.navBarItemColor,
                   ),
                 ]),
-              ),
-            ));
+          ),
+        ),
+      ),
+    );
+  }
+
+  void validateInput(String input) {
```

Comment:
All validation should be in a file

Reply:
@isaacadariku  the requested changes has been made